pkg/controller/globalscylladbmanager/sync_scylladbmanagerclusterregistrations.go

```diff
@@ -72,16 +86,27 @@ func (gsmc *Controller) syncScyllaDBManagerClusterRegistrations(ctx context.Cont
 }
 
 func makeScyllaDBManagerClusterRegistrationsForScyllaDBDatacenters(sdcs []*scyllav1alpha1.ScyllaDBDatacenter) (map[string][]*scyllav1alpha1.ScyllaDBManagerClusterRegistration, error) {
```

Comment:
we keep those in resource.go, lets make it consistent

Reply:
done